comet/core/models.py

```
@@ -673,13 +707,31 @@ def check_debrid_service(cls, v):
     else settings.DATABASE_URL
 )
-database = Database(
-    f"{'sqlite' if settings.DATABASE_TYPE == 'sqlite' else 'postgresql+asyncpg'}://{'/' if settings.DATABASE_TYPE == 'sqlite' else ''}{database_url}"
+
+replica_instances: List[Database] = []
+if settings.DATABASE_TYPE != "sqlite" and settings.DATABASE_READ_REPLICA_URLS:
+    for replica_url in settings.DATABASE_READ_REPLICA_URLS:
+        if replica_url:
+            replica_instances.append(_build_database_instance(replica_url))
+elif settings.DATABASE_TYPE == "sqlite" and settings.DATABASE_READ_REPLICA_URLS:
+    logger.log(
```

Ruff (0.14.8)

Undefined name `logger`

(F821)